### PR TITLE
Rediseño de pagina

### PR DESCRIPTION
**Pull request by FedeMacias**
## Summary
Redesigned the index.html file so it can be styled with grid.
Styled the desktop view using grid.
Created 2 medias, one for cellphone view and one for tablet view, both also using grid.

## Snapshot
rediseño-de-pagina

## Trello
https://trello.com/c/dU3Y9fi7/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -1,362 +1,321 @@
...
+    align-self: center;
+    grid-area: cover-text;
+    height: 416px;
+    width: 900px;
+    background-color: $white;
+    box-shadow: 4px 4px 10px 0 $black-rgba;
+    grid-template-columns: 309px 250px 341px;
+    grid-template-rows: 75px 341px;
+    grid-template-areas:
+    "image titulo genero"
```
Comment (MMarussi) [inline scss/detail.scss]: Remember that anything that it's not bound to be seen by the user should be in english

File: scss/detail.scss [hunk 2]
```diff
@@ -1,362 +1,321 @@
...
+    "image text text";
```
Comment (MMarussi) [inline scss/detail.scss]: Try to use an area name more descriptive than "text"

File: index.html [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
+      <div class="cover-page-badge-container">
+        <img class="cover-page" src="Images/book-cover.png" alt="Wolox books cover page">
+        <img class="badge" src="Images/badge.png" alt="Cerulean and earls green badge with a star in the center">
+      </div>
+      <p class="book-title">
+        Título del libro
+      </p>
+      <p class="book-genre"> (género)</p>
+      <div class="container-text">
+        <div class="container-book">
```
Comment (MMarussi) [inline index.html]: Is this element a line that separates sections?

File: scss/application.scss [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+    border: 2px solid $earls-green;
+    color: $earls-green;
+  }
+}
+
+.master-container {
+  display: flex;
+  height: 768px;
+  width: 100%;
+  max-width: 1366px;
```
Comment (MMarussi) [inline scss/application.scss]: How about creating a variable for this width?

**Review by MMarussi — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (FedeMacias) [reply index.html]: yes, ive realized its just a line that separates sections, i think i could include the border inside the detail container

File: index.html [hunk 5]
```diff
@@ -1,47 +1,48 @@
...
+        </div>
+        <div class="container-detail">
+          <div class="title">
+            Autor del libro:
```
Comment (MMarussi) [inline index.html]: This could be inside a `<p>` tag

File: index.html [hunk 6]
```diff
@@ -1,47 +1,48 @@
...
+            <p class="detail"> Nombre del autor del libro</p>
+          </div>
+          <div class="title">
+            Editorial:
```
Comment (MMarussi) [inline index.html]: Same

File: scss/detail.scss [hunk 7]
```diff
@@ -1,362 +1,321 @@
...
-  
-    .cover-page{
-      display: flex;
-      height: 368px;
-      width: 261px;
-      margin: 24px 52px 24px 24px;
+
+  @media only screen and (max-width: $tablet){
+    height: 768px;
+    width: 1024px;
```
Comment (MMarussi) [inline scss/detail.scss]: $tablet is 1024px, you can use that variable here or even better: Define the rule `max-width` with that value (don't forget the width: 100%)

File: scss/detail.scss [hunk 8]
```diff
@@ -1,362 +1,321 @@
...
-    bottom: 472px;
-    left: 457px;
-    z-index: 1;
-  }
 
-  /* Mobile view */
-  @media only screen and (max-width: 1024px){
+  @media only screen and (max-width: $mobile) {
     height: 667px;
     width: 376px;
```
Comment (MMarussi) [inline scss/detail.scss]: Same as the comment on $tablet

File: scss/detail.scss [hunk 9]
```diff
@@ -1,362 +1,321 @@
...
-        line-height: 24px;
+      .book-genre {
+        margin: 0 0 0 50px;
+        grid-area: genre;
+        justify-self: flex-start;
+        align-self: flex-start;
       }
+
+      .container-text {
+        display: grid;
```
Comment (MMarussi) [inline scss/detail.scss]: Is grid necessary here? There are no template-areas defined nor column/row dimensions :thinking:

File: login.html [hunk 10]
```diff
@@ -1,39 +1,33 @@
...
-    </div>
-    <div class= "button">
-        <a href= "book-list.html" class = "login-button">
-          <p class= "login-text"> Login </p>    
+  <div class="master-container login-master-container">
+    <div class="login-container">
+      <img class="wolox-logo" src="Images/LogoWolox.png" alt="wolox logo">
+      <form class="login-form">
+        <label class="text"> Email </label>
+        <input type="email" class="credentials-box" />
```
Comment (MMarussi) [inline login.html]: Just a suggestion for the future: Keep in mind that assigning a "for" param in the label equal to the "id" of the input it's a good practice (more info on why and how [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)). For example:
```suggestion
        <label for="email" class="text"> Email </label>
        <input type="email" class="credentials-box" id="email" />
```

**Review by mcallegari10 — APPROVED** [review]

**Comment by mcallegari10** [comment]
Remember not to nest everything in SASS, that can and will bring you trouble when overwriting styles and creating a more complex layout

**Pull request closed without merging**